triage/forms.py

```diff
@@ -136,6 +145,26 @@ class CompaniesHouseSearchForm(forms.Form):
     term = forms.CharField()
 
 
+class SectorForm(forms.Form):
+    sector = forms.ChoiceField(
+        choices=BLANK_CHOICE_DASH + SECTORS_CHOICES,
+        label='',
+        label_suffix='',
+        widget=forms.Select(attrs={'id': 'js-sector-select'}),
+    )
+
+    def update_triage_sector(self, request, answers):
+        answers.update(self.cleaned_data)
+        answer_manager = TriageAnswersManager(request)
+        answer_manager.persist_answers(answers)
+
+
+def get_sector_label(cleaned_data):
+    for key, label in exred_sector_names.SECTORS_CHOICES:
```

Comment:
what will happen if `cleaned_data` contains a value that is not in `exred_sector_names.SECTORS_CHOICES`.

passing in `cleaned_data` but accessing only `sector` is a but like asking for a banana but getting a gorilla holding a banana (I love that analagy). Can you pass in `sector` instead of `cleaned_data`?